fallback_formatter.module

```diff
@@ -74,36 +74,51 @@ function fallback_formatter_get_possible_formatters($field_type) {
  * @return string
  *   Rendered HTML of the settings form part.
  */
-function theme_fallback_formatter_settings_order($variables) {
+function theme_fallback_formatter_settings_order(array $variables) {
```

Comment:
Considering how much we're changing here now anyway, maybe it would be easier (to review) to just convert this into a #type table and drop the theme function?